src/perception/image_capture/image_capturepy.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
```

Comment:
what is this script for?

Reply:
That script is used for taking pictures through the zed. Could probably be removed from this commit because it has certain things specific to my machine.

Reply:
maybe just remove your personal file path and move it to the scripts folder

Reply:
Theres also an import `from pynput.keyboard import Key, Controller` that I think is a package specific to my machine. Its necessary to kill the thread, and I'm not sure how you feel about leaving that in here.